common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/MicrosoftAccount.java

```diff
@@ -107,8 +107,9 @@ public MicrosoftAccount(@NonNull final IDToken idToken,
 
         long mPasswordExpiration = 0;
 
-        if (!StringExtensions.isNullOrBlank((String) claims.get(AzureActiveDirectoryIdToken.PASSWORD_EXPIRATION))) {
-            mPasswordExpiration = Long.parseLong((String) claims.get(AzureActiveDirectoryIdToken.PASSWORD_EXPIRATION));
+        if (null != claims.get(AzureActiveDirectoryIdToken.PASSWORD_EXPIRATION)) {
```

Comment:
It might be better doing something like this, so that we perform claims.get() only once

final Object expiry = claims.get(AzureActiveDirectoryIdToken.PASSWORD_EXPIRATION);
if (expiry != null) {
    mPasswordExpiration = Long.valueOf(expiry.toString());
}

Reply:
Like this feedback and incorporated into another part of the same method.

Reply:
Resolved in 83e6e51e3694a8d0a80bf3b1f039f16a32770899